edgraph/access_ee.go

```diff
@@ -351,7 +351,13 @@ func RefreshAcls(closer *z.Closer) {
 		if kvs == nil || len(kvs.Kv) == 0 {
 			return
 		}
-		if err := retrieveAcls(kvs.Kv[0].Version); err != nil {
+		var maxVs uint64
+		for _, kv := range kvs.GetKv() {
+			if kv.Version > maxVs {
+				maxVs = kv.Version
+			}
+		}
+		if err := retrieveAcls(maxVs); err != nil {
```

Comment:
Why were we using the `KV[0]` ? Confirm if we needed the `kv` with the minimum version instead.

Reply:
We want the latest version.